easybuild/easyconfigs/h/HIP/HIP-4.5.0-GCCcore-11.2.0-amd.eb

```
@@ -0,0 +1,93 @@
+easyblock = 'CMakeMake'
+
+name = 'HIP'
+version = '4.5.0'
+versionsuffix = '-amd'
+
+homepage = 'https://github.com/ROCm-Developer-Tools/HIP'
+description = """HIP is a C++ Runtime API and Kernel Language that allows
+developers to create portable applications for AMD and NVIDIA GPUs from single
+source code."""
+docurls = ['https://rocmdocs.amd.com']
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+source_urls = [
+    "https://github.com/ROCm-Developer-Tools/hipamd/archive/",
+    "https://github.com/ROCm-Developer-Tools/ROCclr/archive/",
+    "https://github.com/ROCm-Developer-Tools/HIP/archive/",
+    "https://github.com/RadeonOpenCompute/ROCm-OpenCL-Runtime/archive/",
+]
+sources = [
+    'hipamd-rocm-%(version)s.tar.gz',
+    'ROCclr-rocm-%(version)s.tar.gz',
+    'HIP-rocm-%(version)s.tar.gz',
+    'ROCm-OpenCL-Runtime-rocm-%(version)s.tar.gz',
+]
```

@nordmoen Letting EasyBuild download these sources doesn't work as it is specified name, because the filename to use is the same for all of these.

That can be fixed with a bit of tweaking though:

```suggestion
sources = [
    {
        'source_urls': ['https://github.com/ROCm-Developer-Tools/hipamd/archive/'],
        'download_filename': 'rocm-%(version)s.tar.gz',
        'filename': 'hipamd-rocm-%(version)s.tar.gz',
    },
    {
        'source_urls': ['https://github.com/ROCm-Developer-Tools/ROCclr/archive/'],
        'download_filename': 'rocm-%(version)s.tar.gz',
        'filename': 'ROCclr-rocm-%(version)s.tar.gz',
    },
    {
        'source_urls': ['https://github.com/ROCm-Developer-Tools/HIP/archive/'],
        'download_filename': 'rocm-%(version)s.tar.gz',
        'filename': 'HIP-rocm-%(version)s.tar.gz',
    },
    {
        'source_urls': ['https://github.com/RadeonOpenCompute/ROCm-OpenCL-Runtime/archive/'],
        'download_filename': 'rocm-%(version)s.tar.gz',
        'filename': 'ROCm-OpenCL-Runtime-rocm-%(version)s.tar.gz',
    },
]
```